subxt/src/backend/chain_head/mod.rs

```diff
@@ -543,7 +543,8 @@ impl<T: Config + Send + Sync + 'static> Backend<T> for ChainHeadBackend<T> {
 
     async fn stream_all_block_headers(
         &self,
-    ) -> Result<StreamOfResults<(T::Header, BlockRef<T::Hash>)>, Error> {
+        _hasher: T::Hasher,
```

Comment:
Why do we need pass in the hasher here?

It's already part of T: Config?

Reply:
The hasher interface changed so that you can instantiate a hasher with some metadata (allowing it to decide how to hash based on the metadata). Because of this, we now need to construct a hasher to use (rather than use a static method on an empty hasher type), so we pass this hasher around in order to avoid constructing it as much as possible